.github/workflows/e2e-tests.yml

```diff
@@ -75,4 +75,4 @@ jobs:
         if: ${{ always() }}
         run: |
           set +e
-          az group delete --name "${{ env.CLUSTER_NAME }}" --yes --no-wait || true
+          az group delete --name "${{ env.CLUSTER_NAME }}" --yes --no-wait || true
```

Comment:
nit: add new line.

Reply:
Still complains if I add new line, master branch does not have a new line